setup.cfg

```diff
@@ -0,0 +1,65 @@
+[metadata]
+name = pyispyb
+author = ISPyB collaboration
+author_email = ispyb-dev@esrf.fr
+description = ISPyB FastAPI server
+license_file = LICENSE
+keywords = synchrotron, data acquisition, real-time monitoring, LIMS
+long_description = file: README.md
+long_description_content_type = text/x-markdown
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Operating System :: OS Independent
+    Topic :: Software Development :: Libraries :: Python Modules
+project-urls =
+    download = https://github.com/ispyb/py-ispyb
+    documentation = https://ispyb.gitlab-pages.esrf.fr/py-ispyb/
+
+
+[options]
+python_requires = >=3.9
```

Comment:
I think we need Python >= 3.10 as we use some of the Python 3.10 notation for typing in the code

Reply:
yep good spot